miniapps/electromagnetics/butterfly_3d.mesh

```diff
@@ -0,0 +1,13970 @@
+MFEM mesh v1.0
```

Comment:
This mesh doesn't seems to be used, do we need to add it in?

Reply:
No, we don't.  I was using that for testing and forgot that it was still there.